hazelcast/src/main/java/com/hazelcast/spi/CanCancelOperations.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.spi;
+
+import com.hazelcast.nio.Address;
+
+/**
+ * Implemented by a service that can cancel its operations.
+ */
+public interface CanCancelOperations {
```

Comment:
This interface is actually symmetrical to `LiveOperationsTracker` and should be implemented by the same classes that implement that one. However these implementations are out of scope in this PR. The primary motivation is cancellation of Jet jobs and we can gradually add more cancelability to main HZ operations.